packages/forms-web-app/src/dynamic-forms/controller.js

```diff
@@ -134,8 +135,18 @@ exports.list = async (req, res) => {
 
 			// default question format
 			const key = question.title ?? question.question;
-			const value =
+			const currentQuestion =
 				question.altText ?? journey.response?.answers[question.fieldName] ?? 'Not started';
+			let value = '';
+
+			if (currentQuestion.uploadedFiles) {
+				for (const item in currentQuestion.uploadedFiles) {
+					value = value + currentQuestion.uploadedFiles[item].originalFileName + '</br>';
+				}
+			} else {
+				value = capitalize(currentQuestion);
```

Comment:
capitalization is to ensure yes/no values being pulled through are displayed with the correct case as pointed out by Steve